### ISSUE TITLE
Add theme and frontend uri to organizations

### ISSUE THREAD

**Issue by oculos** [keycloak/keycloak#31268]
### Description

Since organizations are based on domain names, it’d be really nice to have some realm settings available for organizations as well. On the top of my mind, feontend url and theme would be amazing to have on an organization level. 

### Discussion

https://github.com/keycloak/keycloak/discussions/19054#discussioncomment-10037870

### Motivation

Multiple domains might mean different ux. It’d be nice to have more customization options for organizations.

**Comment by ieugen** [comment]
Would love to see this feature.

**Comment by pedroigor** [comment]
Yeah, it is a nice capability and it is one of the tasks we have in https://github.com/keycloak/keycloak/issues/30229.

However, today we have basic support for branding/themes that might help with customizations. In summary, we are exposing beans to the freemarker templates so that you can access the organization resolved when the user is authenticating. See:

* https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationThemeTest.java
* https://github.com/keycloak/keycloak/tree/main/testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/resources/theme/organization/login

Does it help you?

What we have today is basically leveraging the existing mechanisms to customize themes but I can see improving this by automatically selecting a theme based on the organization alias so that you just deploy the theme using a specific directory format.

Regarding the frontend URL, I'm not sure as it will also require reviewing the hostname provider. But I can see the motivation and it makes sense to me as well.
